catalog/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path, include
+from catalog.apps import CatalogConfig
+from catalog.views import home, contacts, index
+
+app_name = CatalogConfig.name
+
+urlpatterns = [
+    path('', index, name='index'),
+    path('home/', home, name='home'),
```

Comment:
Правильнее сделать так:
```suggestion
    path('home/', index, name='home'),
```
Тогда не нужно будет делать дополнительный контроллер (они у тебя идентичны). 
В данном модуле ты лишь говоришь по какому пути какой контроллер будет обрабатывать данные. Поэтому на два разные пути мы можем повесить один контроллер - это не ошибка